### PR TITLE
Jonathan Jamel Holloway

### PR DESCRIPTION
**Pull request by codejoncode**

### CODE REVIEW

**Comment by codejoncode** [comment]
- [ ] fizzbuzz
- [ ] strings
- [ ] pointers
- [ ] malloc
- [ ] structs
- [ ] queue
- [ ] quicksort
- [ ]  [Stretch Goal] hangman

File: fizzbuzz/fizzbuzz.c [hunk 1]
```diff
@@ -10,7 +10,27 @@
...
+        else if (x % 5 == 0)
+        {
+            printf("Buzz\n");
+        }
+        else
+        {
+            counter++;
+        }
+    }
+    return counter;
```
Comment (wvandolah) [inline fizzbuzz/fizzbuzz.c]: 👍

File: strings/strings.c [hunk 2]
```diff
@@ -9,7 +9,17 @@
...
 {
-
+  int length = 0;
+  int i = 0;
+  //initialize both length and i at zero 
+  
+  
+  for(i = 0; s[i] != '\0'; i++){
+    length++;
+  }
```
Comment (wvandolah) [inline strings/strings.c]: `i` and `length` are basically the same thing. you could do this in a while loop or 
```
{
  int length = 0;
  for(; s[length] != '\0';){
    length++;
  } 
  return length;
} 
```

File: pointers/pointers.c [hunk 3]
```diff
@@ -8,7 +8,11 @@
 */
 void swap(int* a, int* b)
 {
-
+  int holdb =*b;
+  int holda = *a;
+  
+  *b = holda;
+  *a = holdb;
```
Comment (wvandolah) [inline pointers/pointers.c]:
```
  {
  int holdb =*b;
  
  *b = *a;
  *a = holdb;
}
```

**Pull request closed without merging**